creator-node/src/snapbackSM/peerSetManager.js

```diff
@@ -39,8 +47,10 @@ class PeerSetManager {
 
     for await (const peer of peerSet) {
       try {
-        // TODO: consider using the response
-        await this.determinePeerHealth(peer)
+        const verboseHealthCheckResp = await this.getPeerHealth(peer)
```

Comment:
nit but `getPEerHealth()` and `determinePeerHealth()` are a bit confusingly named, maybe `queryVerboseHealthCheck()` and `determinePeerHealth()`